simplified-opds-core/build.gradle

```diff
@@ -3,8 +3,6 @@ dependencies {
 
   implementation project(":simplified-json-core")
 
-  implementation libraries.commonsLang
-  implementation libraries.googleGSON
```

Comment:
These deps seemed unused. I ran `assemble` and everything worked okay.

Reply:
Glad to see them gone.